tool/tsh/tsh.go

```diff
@@ -559,6 +568,14 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 		appFormatURI, appFormatCA, appFormatCert, appFormatKey, appFormatCURL, appFormatJSON, appFormatYAML),
 	).Short('f').StringVar(&cf.Format)
 
+	// Sessions.
+	sessions := app.Command("sessions", "View and control recorded sessions.").Alias("session")
```

Comment:
Any reason the help message for `tctl sessions` is different from `tsh sessions`?

Reply:
Fixed